go.mod

```diff
@@ -86,7 +86,7 @@ require (
 	github.com/stackrox/external-network-pusher v0.0.0-20210419192707-074af92bbfa7
 	github.com/stackrox/helmtest v0.0.0-20220930104945-c4a3c15e834a
 	github.com/stackrox/k8s-istio-cve-pusher v0.0.0-20210422200002-d89f671ac4f5
-	github.com/stackrox/scanner v0.0.0-20230120022619-96107ead11f0
+	github.com/stackrox/scanner v0.0.0-20230127183442-f5521e941607
```

Comment:
I saw you reverted the tag in `SCANNER_VERSION` should this also be reverted?

Reply:
I will double check that. My intention was to take both from master for now.

We need to bump the SCANNER_VERSION after this https://github.com/stackrox/scanner/pull/1065 is merged, but let's not make too many steps at once.

Reply:
I did `go get -u github.com/stackrox/scanner@master` and `go mod tidy` and committed [1684b59](https://github.com/stackrox/stackrox/pull/4492/commits/1684b59a61dd7d0dd6d6bade3d1ab118437a6a6d). Will deploy it locally again to test that thoroughly.

Reply:
It looks good locally. I will let the CI finish its checks